{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "024ef371",
   "metadata": {},
   "outputs": [],
   "source": [
    "path = \"C:\\code\\\"\n",
    "\n",
    "import glob\n",
    "\n",
    "txtfiles = []\n",
    "for file in glob.glob(path + \"\\\\*.m4a\"):\n",
    "    txtfiles.append(file)\n",
    "\n",
    "for item in txtfiles:\n",
    "    print(item)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "aff579e1",
   "metadata": {},
   "outputs": [],
   "source": [
    "import glob"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "12e22b3c",
   "metadata": {},
   "outputs": [],
   "source": [
    "path = \"C:/Users/USER/OneDrive/Desktop/All JN Projects/Sigma Apu/New folder\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "1f46b031",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "C:/Users/USER/OneDrive/Desktop/All JN Projects/Sigma Apu/New folder\n"
     ]
    }
   ],
   "source": [
    "txtfiles = []\n",
    "for file in glob.glob(path):\n",
    "    txtfiles.append(file)\n",
    "\n",
    "for item in txtfiles:\n",
    "    print(item)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "c1d8fee2",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "'ls' is not recognized as an internal or external command,\n",
      "operable program or batch file.\n"
     ]
    }
   ],
   "source": [
    "!ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "b790f166",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " Volume in drive C is Windows\n",
      " Volume Serial Number is 8A26-4FE7\n",
      "\n",
      " Directory of C:\\Users\\USER\\OneDrive\\Desktop\\All JN Projects\\Sigma Apu\\New folder\n",
      "\n",
      "07/17/2022  10:00 AM    <DIR>          .\n",
      "07/17/2022  10:00 AM    <DIR>          ..\n",
      "07/17/2022  09:56 AM    <DIR>          .ipynb_checkpoints\n",
      "04/20/2022  12:34 PM    <DIR>          Denchmark_BRs-main\n",
      "07/17/2022  10:00 AM             1,828 Untitled.ipynb\n",
      "               1 File(s)          1,828 bytes\n",
      "               4 Dir(s)  303,649,263,616 bytes free\n"
     ]
    }
   ],
   "source": [
    "ls"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "e97785e0",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os \n",
    "my_list = os.listdir('C:/Users/USER/OneDrive/Desktop/All JN Projects/Sigma Apu/New folder/Denchmark_BRs-main/JSonSet/simplejson')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "b19fe637",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "list"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "type(my_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "938dda6e",
   "metadata": {},
   "outputs": [],
   "source": [
    "a = os.listdir('C:/Users/USER/OneDrive/Desktop/All JN Projects/Sigma Apu/New folder/Denchmark_BRs-main/JSonSet/simplejson/albumentations-team+albumentations')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "f41b6618",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['180.json', '186.json', '207.json', '213.json', '215.json', '760.json']"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "b79736a4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'BR': {'BR_id': '180', 'BR_author': 'leoburgy', 'BRopenT': '2019-02-04T10:30:42Z', 'BRcloseT': '2019-02-15T18:08:39Z', 'BR_text': {'BRsummary': 'CLAHE is not working on grayscale images', 'BRdescription': \"\\n <denchmark-h:h2>üêõ Bug</denchmark-h>\\n \\n <denchmark-h:h2>To Reproduce</denchmark-h>\\n \\n Steps to reproduce the behavior:\\n \\n Import an image as a grey-scale image\\n Construct an augmentation with CLAHE\\n Pass the image to it\\n \\n <denchmark-code>from albumentations import CLAHE\\n import cv2\\n \\n aug = CLAHE(p=1)\\n img = cv2.imread('../data/images/ss1_075.png', cv2.IMREAD_GRAYSCALE)\\n img.shape\\n # (256, 256)\\n aug(image=img, p=1)\\n </denchmark-code>\\n \\n <denchmark-code>---------------------------------------------------------------------------\\n error                                     Traceback (most recent call last)\\n <ipython-input-25-6a9d0f41d618> in <module>()\\n       6 img.shape\\n       7 # (256, 256)\\n ----> 8 aug(image=img, p=1)\\n \\n /usr/local/lib/python3.7/site-packages/albumentations/core/transforms_interface.py in __call__(self, **kwargs)\\n      36                     target_function = self._get_target_function(key)\\n      37                     target_dependencies = {k: kwargs[k] for k in self.target_dependence.get(key, [])}\\n ---> 38                     res[key] = target_function(arg, **dict(params, **target_dependencies))\\n      39                 else:\\n      40                     res[key] = None\\n \\n /usr/local/lib/python3.7/site-packages/albumentations/augmentations/transforms.py in apply(self, img, clip_limit, **params)\\n     985 \\n     986     def apply(self, img, clip_limit=2, **params):\\n --> 987         return F.clahe(img, clip_limit, self.tile_grid_size)\\n     988 \\n     989     def get_params(self):\\n \\n /usr/local/lib/python3.7/site-packages/albumentations/augmentations/functional.py in clahe(img, clip_limit, tile_grid_size)\\n     274     if img.dtype != np.uint8:\\n     275         raise TypeError('clahe supports only uint8 inputs')\\n --> 276     img = cv2.cvtColor(img, cv2.COLOR_RGB2LAB)\\n     277     clahe = cv2.createCLAHE(clipLimit=clip_limit, tileGridSize=tile_grid_size)\\n     278     img[:, :, 0] = clahe.apply(img[:, :, 0])\\n \\n error: OpenCV(3.4.3) /Users/travis/build/skvark/opencv-python/opencv/modules/imgproc/src/color.hpp:255: error: (-2:Unspecified error) in function 'cv::CvtHelper<cv::Set<3, 4, -1>, cv::Set<3, -1, -1>, cv::Set<0, 5, -1>, cv::SizePolicy::NONE>::CvtHelper(InputArray, OutputArray, int) [VScn = cv::Set<3, 4, -1>, VDcn = cv::Set<3, -1, -1>, VDepth = cv::Set<0, 5, -1>, sizePolicy = cv::SizePolicy::NONE]'\\n > Invalid number of channels in input image:\\n >     'VScn::contains(scn)'\\n > where\\n >     'scn' is 1\\n </denchmark-code>\\n \\n <denchmark-h:h2>Expected behavior</denchmark-h>\\n \\n <denchmark-h:h2>Environment</denchmark-h>\\n \\n \\n Albumentations version: 0.1.11\\n Python version: 3.7\\n OS (e.g., Linux): MacOS 10.14\\n How you installed albumentations: pip\\n \\n \\t\"}, 'comments': {'comments_0': {'comment_id': 1, 'comment_author': 'leoburgy', 'commentT': '2019-02-04T16:29:37Z', 'comment_text': \"\\n \\t\\tright now we don't support single-channel images. a workaround is described here: <denchmark-link:https://github.com/albumentations-team/albumentations/issues/67>#67</denchmark-link>\\n \\n \\t\\t\"}, 'comments_1': {'comment_id': 2, 'comment_author': 'leoburgy', 'commentT': '2019-02-08T18:53:39Z', 'comment_text': '\\n \\t\\tIt would be a good idea to add this transformation. Especially useful for medical imagery.\\n \\t\\t'}, 'comments_2': {'comment_id': 3, 'comment_author': 'leoburgy', 'commentT': '2019-02-08T18:54:24Z', 'comment_text': '\\n \\t\\t<denchmark-link:https://github.com/ternaus>@ternaus</denchmark-link>\\n  you are welcome with PR\\n \\t\\t'}, 'comments_3': {'comment_id': 4, 'comment_author': 'leoburgy', 'commentT': '2019-02-15T18:08:39Z', 'comment_text': '\\n \\t\\tFixed in <denchmark-link:https://github.com/albumentations-team/albumentations/pull/190>#190</denchmark-link>\\n \\n \\t\\t'}}}, 'commit': {'commit_id': '8d02597c4297963077db628473815849b868942b', 'commit_author': 'Vladimir Iglovikov', 'commitT': '2019-02-15 10:08:09-08:00', 'changed_files': {'file_0': {'file_change_type': 'MODIFY', 'file_Nmethod': 1, 'file_old_name': 'albumentations\\\\augmentations\\\\functional.py', 'file_new_name': 'albumentations\\\\augmentations\\\\functional.py', 'hunks': {'hunk_0': {'Ismethod': 1, 'added_lines': '300,302,303,304,305,306,307,308,309', 'deleted_lines': '300,302,303', 'method_info': {'method_name': 'clahe', 'method_params': 'img,clip_limit,tile_grid_size,8', 'method_startline': '297', 'method_endline': '310'}}}}}}}\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "\n",
    "with open('C:/Users/USER/OneDrive/Desktop/All JN Projects/Sigma Apu/New folder/Denchmark_BRs-main/JSonSet/simplejson/albumentations-team+albumentations/180.json') as f:\n",
    "  data = json.load(f)\n",
    "\n",
    "print(data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "95199979",
   "metadata": {},
   "outputs": [],
   "source": [
    "real = data[\"BR\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "2390e532",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "180\n"
     ]
    }
   ],
   "source": [
    "print(real[\"BR_id\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "id": "cc162e21",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'BRsummary': 'CLAHE is not working on grayscale images',\n",
       " 'BRdescription': \"\\n <denchmark-h:h2>üêõ Bug</denchmark-h>\\n \\n <denchmark-h:h2>To Reproduce</denchmark-h>\\n \\n Steps to reproduce the behavior:\\n \\n Import an image as a grey-scale image\\n Construct an augmentation with CLAHE\\n Pass the image to it\\n \\n <denchmark-code>from albumentations import CLAHE\\n import cv2\\n \\n aug = CLAHE(p=1)\\n img = cv2.imread('../data/images/ss1_075.png', cv2.IMREAD_GRAYSCALE)\\n img.shape\\n # (256, 256)\\n aug(image=img, p=1)\\n </denchmark-code>\\n \\n <denchmark-code>---------------------------------------------------------------------------\\n error                                     Traceback (most recent call last)\\n <ipython-input-25-6a9d0f41d618> in <module>()\\n       6 img.shape\\n       7 # (256, 256)\\n ----> 8 aug(image=img, p=1)\\n \\n /usr/local/lib/python3.7/site-packages/albumentations/core/transforms_interface.py in __call__(self, **kwargs)\\n      36                     target_function = self._get_target_function(key)\\n      37                     target_dependencies = {k: kwargs[k] for k in self.target_dependence.get(key, [])}\\n ---> 38                     res[key] = target_function(arg, **dict(params, **target_dependencies))\\n      39                 else:\\n      40                     res[key] = None\\n \\n /usr/local/lib/python3.7/site-packages/albumentations/augmentations/transforms.py in apply(self, img, clip_limit, **params)\\n     985 \\n     986     def apply(self, img, clip_limit=2, **params):\\n --> 987         return F.clahe(img, clip_limit, self.tile_grid_size)\\n     988 \\n     989     def get_params(self):\\n \\n /usr/local/lib/python3.7/site-packages/albumentations/augmentations/functional.py in clahe(img, clip_limit, tile_grid_size)\\n     274     if img.dtype != np.uint8:\\n     275         raise TypeError('clahe supports only uint8 inputs')\\n --> 276     img = cv2.cvtColor(img, cv2.COLOR_RGB2LAB)\\n     277     clahe = cv2.createCLAHE(clipLimit=clip_limit, tileGridSize=tile_grid_size)\\n     278     img[:, :, 0] = clahe.apply(img[:, :, 0])\\n \\n error: OpenCV(3.4.3) /Users/travis/build/skvark/opencv-python/opencv/modules/imgproc/src/color.hpp:255: error: (-2:Unspecified error) in function 'cv::CvtHelper<cv::Set<3, 4, -1>, cv::Set<3, -1, -1>, cv::Set<0, 5, -1>, cv::SizePolicy::NONE>::CvtHelper(InputArray, OutputArray, int) [VScn = cv::Set<3, 4, -1>, VDcn = cv::Set<3, -1, -1>, VDepth = cv::Set<0, 5, -1>, sizePolicy = cv::SizePolicy::NONE]'\\n > Invalid number of channels in input image:\\n >     'VScn::contains(scn)'\\n > where\\n >     'scn' is 1\\n </denchmark-code>\\n \\n <denchmark-h:h2>Expected behavior</denchmark-h>\\n \\n <denchmark-h:h2>Environment</denchmark-h>\\n \\n \\n Albumentations version: 0.1.11\\n Python version: 3.7\\n OS (e.g., Linux): MacOS 10.14\\n How you installed albumentations: pip\\n \\n \\t\"}"
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "real[\"BR_text\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e6d5866e",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "c35fe3b7",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import json\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "64b7ead7",
   "metadata": {},
   "outputs": [],
   "source": [
    "path = 'C:/Users/USER/OneDrive/Desktop/All JN Projects/Sigma Apu/New folder/Denchmark_BRs-main/JSonSet/simplejson'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "c1385e55",
   "metadata": {},
   "outputs": [],
   "source": [
    "folders = os.listdir(path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "id": "46814ffe",
   "metadata": {},
   "outputs": [],
   "source": [
    "BR_id = []\n",
    "BRsummary = []\n",
    "BRdescription = []\n",
    "Project_name = []"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "04c85eab",
   "metadata": {},
   "outputs": [],
   "source": [
    "for folder in folders:\n",
    "    file_path=path+\"/\"+folder\n",
    "    files = os.listdir(file_path)\n",
    "    for file in files:\n",
    "        full_path=path+\"/\"+folder+\"/\"+file\n",
    "        with open(full_path) as f:\n",
    "            data = json.load(f)\n",
    "        real = data[\"BR\"]\n",
    "        BR_id.append(real[\"BR_id\"])\n",
    "        BRsummary.append(real[\"BR_text\"][\"BRsummary\"])\n",
    "        BRdescription.append(real[\"BR_text\"][\"BRdescription\"])\n",
    "        Project_name.append(folder)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "id": "d92d0f53",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "4577"
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.DataFrame(list(zip(BR_id, BRsummary, BRdescription, Project_name)),columns =['BR_id', 'BRsummary', 'BRdescription'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "id": "5811b622",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "4577"
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(BRsummary)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "id": "56329eac",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['CLAHE is not working on grayscale images',\n",
       " 'RandomScale, ShiftScaleRotate strange \"scale\" aug behavior',\n",
       " 'The output shape of `smallest_max_size` is smaller than the speficified size',\n",
       " 'GaussNoise does not work correctly on small images',\n",
       " 'Error in returning invisible keypoints',\n",
       " 'CropNonEmptyMaskIfExists with multiple masks not working',\n",
       " \"The tutorial content 'Get started with Pipeline API' does not exist.\",\n",
       " 'costa: frequent timeout errors when running the CI tests',\n",
       " 'cli: incorrect cli output when daemon is crashed',\n",
       " 'bug: New Typescript version breaks tsdoc',\n",
       " 'Daemon: throw error  `unexpected end of file` when upload a plugin from tarball.',\n",
       " 'daemon: failed to start daemon',\n",
       " \"build: dev-build doesn't work\",\n",
       " 'training failedÔºåËÆ≠ÁªÉcancel ‰πãÂêéËÆ≠ÁªÉËøõÁ®ãÊ≤°ÊúâÈÄÄÂá∫',\n",
       " 'boa: some unit tests not working on BigSur',\n",
       " '\"Pipcook init\" node-gyp rebuild error',\n",
       " 'Async model checkpoint upload fails when the file is renamed',\n",
       " 'Error when uploading keras models with trains==0.16.3',\n",
       " 'Delete hyper-parameters from web-app',\n",
       " 'Pipeline controller execute_remotely inconsistent',\n",
       " 'Inconsistent epoch numbering',\n",
       " 'Bug in AutoRegressiveSeqDecoder',\n",
       " 'PretrainedTransformerEmbedder does not pass masks to *transformer models',\n",
       " 'Distributed training bug',\n",
       " '\"TypeError: not a sequence\" on simple coreference resolution',\n",
       " 'Peak CPU memory not reported correctly in distributed training',\n",
       " 'Predictor.from_path not loading my trained model',\n",
       " 'Transformer tokenizers cause deadlocks when dataset reader is lazy and dataloader num_workers &gt; 0',\n",
       " 'Bug on calculating \"argmax_predictions\". It increases the false positive count for last label.',\n",
       " \"Can't get correct output using .saliency_interpret_from_json() from Interpret\",\n",
       " \"PDB++ arrow/tab keys don't work with allennlp\",\n",
       " 'Unexpected behavior with DataLoader.batches_per_epoch',\n",
       " 'PretrainedModelInitializer fails to initialize a model with a 0-dim tensor',\n",
       " 'ModuleNotFoundError when num_workers&gt;0 & DistributeDataParallel & lazy read',\n",
       " '[Suggestion] One line change in the comments',\n",
       " 'When extracting an archive fails half-way through, it corrupts the cache',\n",
       " 'After #4470 the output is duplicated and Tqdm effect is lost',\n",
       " '`batches_per_epoch` should not affect evaluation',\n",
       " 'PretrainedTransformerMismatchedIndexer fails silently when given empty strings as input.',\n",
       " 'Cannot train NER model that uses bidirectional_lm_token_embedder: RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu',\n",
       " 'Problem with PretrainedTransformerEmbedder and models such as T5',\n",
       " 'Closing the TensorBoard writer',\n",
       " 'When running multiple distributed runs on the same box, AllenNLP says \"RuntimeError address already in use‚Äù',\n",
       " 'Rouge metric incorrectly computed with distributed training',\n",
       " 'Potential bug: The maxpool in cnn_encoder can be triggered by pad tokens.',\n",
       " 'trainer_callbacks parameter missing from GradientDescentTrainer.from_partial_objects',\n",
       " 'PretrainedTransformerTokenizer fails when disabling \"fast\" tokenizer',\n",
       " 'GradientDescentTrainer breaks when constructed with validation_data_loader==None and learning_rate_scheduler!=None',\n",
       " 'Rename token.py to avoid bugs in certain Python versions',\n",
       " \"ShardedDatasetReader doesn't inherit lazy from base reader\",\n",
       " 'States are not handled correctly when passed to non-stateful Seq2SeqEncoders',\n",
       " 'Cannot save visualization of topic model to custom filepath',\n",
       " '[CMake][IoT][Jetson] Building with CMake produces libomp.so',\n",
       " 'MXNET_MKLDNN_DEBUG=1 produces errors',\n",
       " 'Example SSD prepare dataset toos are incompatible with im2rec in MxNet 1.1',\n",
       " 'why the gluon.nn.embedding always return float32? seems cannot chang to float64.',\n",
       " 'Language model example cannot be run',\n",
       " 'Build fails with USE_CUDNN = 0',\n",
       " 'Elementwise multiplication of two ndarrays returns wrong results',\n",
       " 'Loading an older model with a custom operator from Java/Scala more than twice causes MXNet to crash',\n",
       " 'MXNet operator profile aggregate counter issue',\n",
       " 'python build (with cython) seems broken in linux (python3)',\n",
       " 'make docs on Mac - scala docs failure',\n",
       " 'Scala Module API not resizing properly when returning to original binded size',\n",
       " '_backward_softsign activation is incorrect',\n",
       " 'Broken test_sparse_operator.test_sparse_mathematical_core with scipy 1.1.0',\n",
       " 'Failed dist_sync_kvstore test not detected on CI for a long time',\n",
       " 'wget https://s3-us-west-2.amazonaws.com/mxnet.liuyz/data/mnist/train.txt throw 404',\n",
       " 'The sampling_ratio in roi-align can be specified(not adaptive size)!',\n",
       " 'Undefined Behavior of mx.sym.where with shape-mismatched cond',\n",
       " \"Seems the gperftools-devel and jemalloc-devel wouldn't be detected correctly on rhel(Redhat Enterprise Linux)\",\n",
       " 'ReLU Clips NaNs to Zero',\n",
       " \"mx.nd.zeros/ones don't recognize cpu_shared as context\",\n",
       " 'Conv1D throws CUDNN_STATUS_EXECUTION_FAILED',\n",
       " 'nd.topk regression with nan values',\n",
       " \"gluon bug: AttributeError: '_thread._local' object has no attribute 'value'\",\n",
       " 'compiling error when USE_CUDA=1  and USE_CUDNN=0 with make command',\n",
       " 'CRF weights never updated in Bi-LSTM-CRF',\n",
       " 'Flaky test test_gluon_trainer.test_trainer_reset_kv',\n",
       " \"nd.softmax() doesn't support grad_req='add'\",\n",
       " 'lstm_bucketing example has evenly divide problem with self create dataset',\n",
       " \"Callback's bug with very fast speed.\",\n",
       " 'Autograd fails when using `take` operator repeatedly',\n",
       " 'some big numbers can not be assigned correctly with ndarray, it is a big bug???',\n",
       " 'All the tests in tools/coreml package are failing',\n",
       " 'gluon.SymbolBlock cannot imports resnet trained with dtype=\"float16\"',\n",
       " 'Model inference outputs not matching across MXNet versions using save_params and load_params',\n",
       " 'Python, Bug: Speech_recognition crashes in threaded_engine.h:379: Error: compute_ctc_loss, stat = execution failed',\n",
       " \"Backwards compatibility checker:  Parameter 'model.1._unfused.0.l_cell.i2h_bias' is missing in file 'lstm_gluon_save_parameters_api-params'\",\n",
       " 'Mxnet 1.3 Bug : Prelu activation compution fault in expand_shape function',\n",
       " 'Python list as a gluon Dataset',\n",
       " 'Spelling mistake in \"mxnet/symbol/image.py\" in which \"gen_image\" is written to be \"gen_iamge\"',\n",
       " 'init.Constant does not work in symbol',\n",
       " '[R] use of mx.io.arrayiter completely crashes R environment',\n",
       " 'CI fails consistently in test_tvm_bridge',\n",
       " 'AddVersion.py breaks the site output in a variety of ways',\n",
       " 'Those cases with some problems in dir : example',\n",
       " 'Report a bug of Engine::Get()',\n",
       " 'Interop bug - clojure - executor/forward - param coercion issue',\n",
       " 'Memory leak when passing images of different dimensions with MXNET_CUDNN_AUTOTUNE_DEFAULT',\n",
       " 'Cannot print parameter summary of embedding layer',\n",
       " 'Name conflict when serializing LSTMCell',\n",
       " 'nd.sparse.clip operation return type is different from the input',\n",
       " 'mxnet.image.imdecode crash if input a empty buffer',\n",
       " 'requests need upgrade due to security risk (CVE-2018-18074)',\n",
       " 'UpSampling Operator Documentation is incomplete',\n",
       " 'size_array operator returns wrong results on MacOS',\n",
       " 'MKLDNN softmax outputs NaN in mkldnn 0.14',\n",
       " \"Many operators don't work for integer type\",\n",
       " 'Data type error when import onnx model',\n",
       " 'MXPredCreate lost dtype',\n",
       " 'Running the mxnet scala jar in the interpreter throws an error',\n",
       " \"SymbolBlock doesn't print anything\",\n",
       " 'Converting MX array to DLPack crashes when MX array goes out-of-scope',\n",
       " 'environment variable page no longer accessible',\n",
       " 'Adam, AdaMax and FTML cannot be used with Trainer(update_on_kv=False)',\n",
       " 'nd.slice does not return empty tensor when begin=end',\n",
       " 'Dropout is Slow',\n",
       " \"[1.4.0] unravel_index no longer works with magic '-1' in shape parameter as in 1.3.1\",\n",
       " 'C API should *never* call abort()',\n",
       " 'website redirect loop on error pages',\n",
       " \"Scala Infer ObjectDector crashes on models which don't return\",\n",
       " 'example/image-classification/common/util.py get_gpus() exception',\n",
       " 'Makefile plugins target needs mkldnn dependency',\n",
       " \"Bug in Optimizer's serializeState and deserializeState methods (Scala)\",\n",
       " 'SoftmaxOutput crashes with normalization \"valid\"',\n",
       " 'MXNET_BACKWARD_DO_MIRROR is broken',\n",
       " 'mx.nd.Custom not working in subprocess',\n",
       " 'resnet cpp-package test is broken',\n",
       " 'Updating mxnet from 1.0.0, networks give different outputs',\n",
       " 'mx.random.seed with ctx failures on a gpu build when run with cpu context',\n",
       " 'RandomResizedCrop produces wrong aspect ratios.',\n",
       " 'Scala/Java reshape problem',\n",
       " 'Inconsistent naming of pow/power between mx.symbol and mx.nd',\n",
       " 'upcasting accumulation type in norm increases loss/Perplexity',\n",
       " '[Numpy][Keras-MXNet] Infer shape partial failed for unknown shapes',\n",
       " 'add_n operator with MXNet-MKL producing wrong results when input count &gt;4',\n",
       " '[MXNet] - [BERT]',\n",
       " 'MXNet to ONNX export bug',\n",
       " '[DOC] Build from source link for ubuntu is broken',\n",
       " 'The discription of function mxnet.image.imresize is wrong.',\n",
       " 'mx.nd.random.shuffle crashes in the master branch',\n",
       " 'Bug in BidirectionalCell for imperative mode when time_step is 1',\n",
       " \"mx.nd.contrib.boolean_mask doesn't support GradientAddition\",\n",
       " '[Bug] Temporary autograd.Function object causes key deletion error',\n",
       " 'ImageRecordIter interpolation method (inter_method) fails for NN (0)',\n",
       " 'Performing operations on 1-D CSR NDArray results in exception',\n",
       " \"Backward doesn't work on LSTM with sequence_length\",\n",
       " \"mxnet.io.NDArrayIter does not properly function when len(data) * 2 &lt; batch_size and last_batch_handle='pad'\",\n",
       " 'Bug in NAG Optimizer',\n",
       " 'Benchmark/opperf on CPU',\n",
       " 'Using openblas',\n",
       " 'Custom Operator Seg Fault',\n",
       " \"Regression in nightly: TypeError: squeeze() missing 1 required positional argument: 'axis'\",\n",
       " 'Transpose does not support inplace',\n",
       " 'Increase amp support for Bi-lstm and Concat operators in gluon',\n",
       " '[Optimizer][Bug] Gradient is mutated in the Adam optimizer',\n",
       " 'FullyConnected op with float64 and MKL-DNN fails if gradient are not set in a specific way',\n",
       " 'TBlob bug about dltensor',\n",
       " 'Out of bounds access in req vector in MKLDNN',\n",
       " 'LSTM with MKL-DNN produces wrong output after weights are changed',\n",
       " '[Bug] Failed to evaluate gradient on samples with train_mode=False',\n",
       " 'Reduce op throws \"too many resources requested for launch\"',\n",
       " '`lein deps` on `contrib/clojure-package/examples/gan` fails',\n",
       " 'cudaErrorInvalidResourceHandle error when running some RNN models',\n",
       " 'cuDNN RNN dtype_with_fallback_ calc needs update',\n",
       " 'Example link of the image classification show 404',\n",
       " '[Numpy] Cannot mix numpy scalar and MXNet numpy ndarray',\n",
       " '[Numpy] Zero-size tensor add zero-size sum raises exception',\n",
       " 'mx.io.NDArrayIter cant pad when size is large',\n",
       " 'Built from source uses old version of opencv',\n",
       " 'gamma and beta dimensionality error in GroupNorm operator',\n",
       " '[Numpy] where does not support python scalar as the input',\n",
       " '[Numpy] np.broadcast_to does not support scalar as input',\n",
       " 'Compile from Source instructions re-use CI install scripts',\n",
       " 'Cmake with NCCL flag does not work.',\n",
       " 'Incorrect installation instruction',\n",
       " 'Graph Partition API Test Error',\n",
       " 'Error when building mxnet with USE_DIST_KVSTORE',\n",
       " 'Error in mkldnn unit test',\n",
       " 'Indexing ndarray should give empty array output',\n",
       " 'Memory corruption issue with unravel_index',\n",
       " 'C++-Package examples broken',\n",
       " 'np.random.gamma(1,0) and np.random.gamma(0,0) hang',\n",
       " 'CMake GPU build broken by #16654',\n",
       " 'Fail to import MXNet in distributed kvstore test',\n",
       " 'MKLDNN incompatibility with large tensor (dim &gt;= 2^32) data',\n",
       " 'Missing dnnl_config.h when installing horovod',\n",
       " '[Numpy] unknown type_flag=7',\n",
       " '[OPPERF] opperf error out if I use python timer instead of the builtin mxnet profiler',\n",
       " '[LayerNorm] Missing the mismatch cues of in_channels',\n",
       " 'cmake stuck in infinite loop at configuration if nvcc is not found',\n",
       " '[Numpy] Symbolic basic indexing bug',\n",
       " '[Website 2.0] General Version Dropdown',\n",
       " 'ImageNet Inference: GPU',\n",
       " '[Numpy] Cannot mix int32 and int64',\n",
       " 'USE_TVM_OP=1 build broken with DMLC_LOG_FATAL_THROW=0',\n",
       " '[Bug][Numpy] Wrong gradient of np.where',\n",
       " 'MKLDNNConvolutionBackward accesses out of bound elements',\n",
       " 'Wrong result when using new numpy ffi in deferred compute',\n",
       " 'Typo on tutorial website',\n",
       " 'CI: cran broken',\n",
       " '[Numpy] Dense does not support float32 + float64.',\n",
       " 'config/linux.cmake  - misleading comment about setting Debug build',\n",
       " 'Error in test_contrib_amp.py::test_amp_coverage',\n",
       " '[Numpy] The gradient of einsum is wrong',\n",
       " '[Numpy] np.split will trigger an error',\n",
       " 'caffe test activation or removal',\n",
       " 'Compiling with USE_TVMOP flaky due to downloading',\n",
       " '[Numpy] np.random.normal/gumbel/logistic broken after FFI update',\n",
       " 'Nightly test fail with \"test_large_vector.test_nn\" on v1.7.x branch',\n",
       " 'same calculation for CrossEntropy and NegativeLogLikelihood in mxnet.metric',\n",
       " 'Divergence of implementation between Windows and non-Windows',\n",
       " 'CD: Missing openblas',\n",
       " 'Fail to use \"==\" operation between different types',\n",
       " 'test_recordimage_dataset',\n",
       " 'Type inference bug of negation',\n",
       " '[Numpy] np.where fails for scalar',\n",
       " \"Can't install Mxnet on raspberry pi with opencv 4.0.0\",\n",
       " 'Matchall in visualize.jl has been deprecated as of Julia 1.3',\n",
       " 'Deferred computation API returns \"wrong\" data type given DeepNumpy input',\n",
       " 'Deferred computation does not work in HybridSequential',\n",
       " 'TestNode.test_exports fails on macOS following \"Implement ONNX export for gluon LSTM\"',\n",
       " '[Website] Java, Scala, C++, R, Clojure, and Julia API References redirect to 404',\n",
       " 'MXNet serialization format depends on endianness',\n",
       " '[Doc] Gluon Block/HybridBlock/SymbolBlock appears at the wrong place in the API doc',\n",
       " 'mx.symbol.numpy._Symbol.__deepcopy__ logic error',\n",
       " 'Transpose only supports up to 6 dimensions',\n",
       " '[Numpy] Backward of softmax, logsoftmax failed on empty ndarray',\n",
       " 'Auto-encoder example out of date',\n",
       " 'metric.py change between branches - missing docs in master',\n",
       " 'executor_manager doc shows up blank in master',\n",
       " 'Numpy Op fallback does not work for functions without return value',\n",
       " 'bfloat16 rendered as uint16',\n",
       " 'MacOS nightly build broken',\n",
       " 'contextvar dependency for MXNet nightly build',\n",
       " 'Default CPU allocator no longer multiple of 64 bytes',\n",
       " 'Website API doc is empty',\n",
       " 'Image transforms sometimes produce incorrect result',\n",
       " \"Can't link master version docs\",\n",
       " 'Floating Point Exception mxnet.ndarray.contrib.bipartite_matching',\n",
       " 'Floating point exception in mxnet.ndarray.InstanceNorm',\n",
       " 'Documentation bug: mxnet.test_utils.check_consistency  Inconsistent parameter',\n",
       " 'Numpy Equal Broadcast Type Large Int Bug',\n",
       " 'MKLDNN numpy rnn core dump',\n",
       " '[Bug] RTC Failed to compile',\n",
       " '[Bug][Numpy] MXNet fp16 initialization bug',\n",
       " 'Multiple subgraph properties with single backend issue with optimize_for',\n",
       " 'infer shape error in hybridized block for zero-size input',\n",
       " '[CI Infrastructure] R-MKLDNN-CPU test run failure.  gcc fails due to ccache issue?',\n",
       " 'Variable sequence length not handled correctly for BiDirectional layers',\n",
       " 'The print output of `mx.nn.Conv2dTranspose` shows the wrong direction of in and out channels.',\n",
       " \"what's wrong with PR test?\",\n",
       " \"net.optimize_for doesn't work with numpy semantics\",\n",
       " '[Bug][AMP][2.0] AMP issue of the concatenate operator',\n",
       " 'pytest xdist internal error',\n",
       " 'CI slowdown on CentOS CPU related to OpenMP and intgemm',\n",
       " 'CheckPvalGPU undefined in CPU builds',\n",
       " 'pip install mxnet-cu102==1.6.0 error ÂÆòÁΩë',\n",
       " '[Bug] Serialization Bug',\n",
       " 'Edge pipeline segmentation fault',\n",
       " \"CI: rat check passing with 'java: command not found' on master\",\n",
       " '`Distribution` repr fails if cached_property not initialized',\n",
       " 'Shape of labels does not match shape of predictions',\n",
       " 'training-accuracy nan',\n",
       " 'mxnet.rnn.BucketSentenceIter has bug',\n",
       " 'Build error: \"‚Äòcv‚Äô does not name a type\" with USE_OPENCV = 0',\n",
       " 'Links on the page with the RNN tutorials are broken',\n",
       " 'Call symbol/nd.smooth_l1 without scalar causing uncaught exception',\n",
       " 'accuracy of cpp example is a constant value when training, no matter how many epochs trained!',\n",
       " 'Bug: Shape of one dimension lead to crash in LogLoss::Update',\n",
       " 'Enable the test test_ndarray.test_cached after fixing the crash.',\n",
       " '[R] Bug introduced in initializer between 0.10.1 and 0.11.1',\n",
       " 'topk/mshadow::range returns wrong result',\n",
       " 'test_arange failure - Jetson TX2 (CPU)',\n",
       " 'Cast storage to same stype gives an error for csr',\n",
       " 'Fatal JVM Error due to Exception in CustomOpProp#inferTypeEntry',\n",
       " '[gluon] segmentation error when getting the shape of output ndarray of transpose conv',\n",
       " 'Run \\'fine-tuning\\' example found„ÄêWrong number of inputs„Äëon \"python3.5 mxnet 0.11.0 centos7 \"',\n",
       " 'Understanding the correlation layer',\n",
       " 'got segfault from lenet with stn example',\n",
       " 'sample_multinomial crashes on vector input',\n",
       " 'Cannot continue with MXNet Tutorial: Real-time Object Detection with MXNet On The Raspberry Pi',\n",
       " 'Segfault on ndarray with negative dimension i.e. mxnet.nx.zeros((-1,))',\n",
       " 'Loss of Precision in BatchNorm and output_var may be wrong',\n",
       " 'test_operator.test_laop_3 hangs',\n",
       " 'nd.pick core dumps on large array',\n",
       " 'Incorrect weight_decay implementation in AdaGrad',\n",
       " 'BucketingModule causes uncaught exception if symbol name not specified',\n",
       " 'SGLD Optimizer in python cannot work due to wrong argument position',\n",
       " 'problem when set fix_gamma=True in batchnorm',\n",
       " 'Android build broken',\n",
       " 'BatchNorm Evaluation Mode Backward Fails with cudnn Enabled',\n",
       " 'BUG in MultiBoxTargetForward when there is single box label',\n",
       " 'from tensorboard import SummaryWriter wrong',\n",
       " 'ndarray indexing issues',\n",
       " '[R] mx.io.arrayiter shuffing is disabled',\n",
       " 'C++ Convolution layer usage produces \"Segmentation fault\"',\n",
       " 'Flaky test_gluon_model_zoo_gpu.test_training @ Python3: MKLDNN-GPU',\n",
       " 'Custom Function Shape Inference',\n",
       " 'Imperative regression output layers are broken',\n",
       " 'MxNet allow to use same name',\n",
       " 'Asynchronous Issue on CustomOP and mxnet.image.ImageDetIter',\n",
       " 'The assignment problem about NDArray',\n",
       " 'fcntl on windows',\n",
       " 'Supporting reduction domains where the RDom depends on axis variables?',\n",
       " '[TEXPR][PASS] Loop distribution pass generates incorrect code',\n",
       " '[RELAY] Avoid eager creation of global target object',\n",
       " '[TOPI] Operator overloading issue when dealing with zero-rank tensor',\n",
       " '[ROCM] Failed topi unittests for rocm backend',\n",
       " '[Bug][Relay] Winograd Conv2D for auto-scheduler errors out when input image is not square',\n",
       " 'Latest coremltools not compatible with Tensorflow. 2.4.0-rc0',\n",
       " 'Model Visualization: layer names do not match exactly',\n",
       " 'Setup.py should list Linux as a supported OS and include copyright header',\n",
       " '\"RuntimeError: failed to convert from IR to Core ML.\" error when converting tf.keras to coreml in tensorflow 2.0',\n",
       " 'Deprecation warning due to invalid escape sequences and literal comparison in Python 3.7',\n",
       " 'cuda tensor parameter fail to convert to numpy in InternalTorchIRGraph',\n",
       " 'PyTorch to CoreML LSTM model conversion v4 fails with ValueError: Layer with name \"_lstm_h0_reshaped_expanded\" has already been added. Please use a unique name.',\n",
       " \"Todays' beta2 checkin breaks  old  issue #759  F.max_pool2d where stride is NOT explicitly specified\",\n",
       " 'Setup.py missing \"typing\" dependency for Python 2.7',\n",
       " 'In Pytorch conversion ConvTransposed with a weight matrix ndims!=4 not working',\n",
       " 'Conversion fails with PyTorch v1.7.0',\n",
       " 'PyTorch topk not supported after mention in release notes',\n",
       " 'Model download should check return code',\n",
       " 'Check failed: l == 1 || r == 1 operands could not be broadcast together with shapes [416,416,4] [1,1,3]',\n",
       " 'SArray.show() is not working in Jupyter Notebook',\n",
       " 'VisionFeaturePrint - Screen or Scene?',\n",
       " 'Update Readme now that v5.0 is out of beta',\n",
       " 'Image deep feature extractor vulnerable to invalid model cache',\n",
       " 'Unintuitive Behaviour in SArray dtype specification',\n",
       " 'Issue with \"turicreate. item_content_recommender\"',\n",
       " 'astype(None) swallows low-level C exception.',\n",
       " 'Tuples get parsed as arrays which are unhashable',\n",
       " 'object_detector.create operands could not be broadcast together',\n",
       " 'SFrame filter_by should work with map data type',\n",
       " \"UG relative links don't work well on apple.github.io/\",\n",
       " 'Double negative in error message',\n",
       " 'Error message for unsupported platform is not obviously from Turi Create',\n",
       " 'Recommender Exclude Parameter Not Working',\n",
       " 'Scatter plot and heat map start axes at 0',\n",
       " 'Unclear error message when calculating jaccard and weighted_jaccard distance',\n",
       " 'SFrame.read_json() treats boolean values as strings',\n",
       " 'NameError in text_analytics.tokenize',\n",
       " 'Image not show on Google Colab Jupyter Notebook',\n",
       " 'Linear Regression Gives Incorrect Error Message with Image Column',\n",
       " 'Object Detector deployed to Vision framework unexpectedly yields confidences at 0.999',\n",
       " 'Failures in CoreMLExportTest due to undefined behavior (observed on 10.13)',\n",
       " 'Configure break on older CMake',\n",
       " 'Size of Recommender Model',\n",
       " 'Exception when importing TuriCreate on a machine with no GPUs',\n",
       " \"Per-class accuracy doesn't make any sense\",\n",
       " 'turicreate.show  bad_get error',\n",
       " 'Logistic Classifier: disable probability output type for multiclass predict',\n",
       " 'Remove references to blas and lapack',\n",
       " 'SFrame.show crashes if there are 0 effective columns',\n",
       " 'Image classifier Error in python3.6',\n",
       " \"Annotation UI doesn't work on an SFrame with more than one image column\",\n",
       " 'Image Classifier Evaluation has Hard Coded Image Column Name',\n",
       " 'Integer labels still create annotations with type str',\n",
       " 'Error message for `.annotate` on a mistyped feature column should be more descriptive',\n",
       " 'Model is hardcoded to \"Resnet50\" in Model Evaluation UI',\n",
       " \"Model Evaluation Doesn't Steal Focus\",\n",
       " 'Int labels cause crash in Evaluation.explore()',\n",
       " 'Clicking in model understanding UI causes a weird selection effect',\n",
       " 'API docs missing for `.annotate`',\n",
       " 'Histogram ‚ÄúCount‚Äù vs. ‚ÄúValues‚Äù is reversed',\n",
       " 'Histogram of ints should have integer bin boundaries',\n",
       " 'read_csv with mismatched quote characters fails silently.',\n",
       " 'In JSON, explicitly quoted \"false\", \"true\" turn to integer 0, 1',\n",
       " 'Creating a Sound Classifier for Short Audio Clips',\n",
       " 'Intermittent Python test failure in test_audio_functionality.py',\n",
       " 'RuntimeError.message is invalid in Python 3, replace in _',\n",
       " 'Lapack not properly detected on build',\n",
       " 'Sketch.num_na() is broken for float',\n",
       " 'Wrong behavior for nan converting from float to int',\n",
       " \"Cmd+Q doesn't quit Turi Create Visualization\",\n",
       " 'CoreML export of boosted tree classifier gives wrong predictions',\n",
       " 'Delay from .show until window appears in the foreground',\n",
       " 'Process crash with wrong flexible_type::get&lt;&gt;',\n",
       " \"Bounding box errors and troubleshooting when using 'explore'\",\n",
       " 'C-API of Activity Classifier gives error when predict is called without giving class labels column',\n",
       " 'Drawing Classifier does not show validation loss while training',\n",
       " 'SFrameBuilder Segmentation Fault on appending incorrect number of columns',\n",
       " 'Annotations should error out on empty Frame',\n",
       " '.explore and .annotate both hang when an image column contains a missing value',\n",
       " 'explore() incorrectly concatenates array elements',\n",
       " 'Can train a model with single class in Sound Classifier',\n",
       " 'Evaluation UI shows incorrect number of errors',\n",
       " 'Drawing classifier evaluate().explore() throws JSON error',\n",
       " \"Annotations when column 'annotations' already exists\",\n",
       " 'Type check for value of max_iterations',\n",
       " 'question: style transfer',\n",
       " 'Single Unit Test for One Shot Object Detection Takes Longer than Ten Minutes',\n",
       " 'Error bundling dylib and pushing to App Store',\n",
       " 'Unclear error message in apply method when most values are None',\n",
       " 'SAarray fails to construct from data when dtype is explicitly specified',\n",
       " 'MAC - Turicreate tc.object_detector.util.draw_bounding_boxes error',\n",
       " 'Travis 3.6 builds are tested on a 2.7 machine',\n",
       " 'Reconcile Vega.js versions and upgrade to Vega 5',\n",
       " 'Breaks compatability with click library.',\n",
       " 'One Shot Object Detector has high disk footprint.',\n",
       " '`dropna` followed by `unpack` is not lazy.',\n",
       " 'Internal groupby does not use atomics where appropriate',\n",
       " 'dict_keys and dict_values types do not work with filter_by',\n",
       " 'unity_sframe::append should accept SFrame with different name order',\n",
       " 'Illegal instruction (core dumped) error after Turicreate version 5.4',\n",
       " 'csv header parsing problem with BOM UTF8',\n",
       " 'HTTPS downloads fail with Turi Create 5.7',\n",
       " 'segfault when dropping words in text_classifier on table containing undefined values.',\n",
       " 'Object Detection Linux build failure',\n",
       " 'Object Detection C++ Test Failure',\n",
       " '`debug/test` build failure.',\n",
       " 'Document which platforms turicreate is able to support GPU acceleration.',\n",
       " 'Drawing classifier validation set to None should work',\n",
       " 'Classifier evaluation should have an explore in 6.0',\n",
       " 'Tensorflow prints out strange text',\n",
       " 'Upgrade the version number of the Python egg to 6.0',\n",
       " 'Setting number of GPUs to 0 on a CUDA machine should disable use of GPUs',\n",
       " 'Activity classifier adds training/validation loss to the display',\n",
       " 'turicreate.load_audio broken with home directory present.',\n",
       " 'Drawing classifier crashes on master with no validation set option',\n",
       " 'OD predict should return bounding box coordinate in normal image scale.',\n",
       " 'Never call tensorflow.compat.v1.reset_default_graph()',\n",
       " 'OD output regression for evaluation.',\n",
       " 'Error comparing  recommender models',\n",
       " 'export_coreml does not respect ~ in filenames (sound classifier).',\n",
       " 'od 6.0 consumes weights exported from 5.8 but has inconsistent evaluation result',\n",
       " 'Style Transfer stylize does no parameter checking',\n",
       " 'output regression for OD export_coreml meta data',\n",
       " 'Style Transfer model loads lazily',\n",
       " 'Style Transfer: model.__repr__() on model loaded from 5.8 throws error',\n",
       " \"OD predict function shouldn't depend on the annotation column\",\n",
       " 'OD: invalid coordinate value gets no error and make it through',\n",
       " 'AC: Classify still go through for data with missing feature column',\n",
       " '[6.1] AC: Strange behavior when k &lt;= 0 for predict_topk. Should check for the input.',\n",
       " \"IOError: [Errno 13] Permission denied: '/var/tmp/model_cache/resnet-50-symbol.json'\",\n",
       " \"All toolkit's summary function should throw exception for invalid input.\",\n",
       " \"SC's strange error message after a model is created.\",\n",
       " '[SC] Invalid custom_layer_sizes should error out.',\n",
       " '[SC] invalid batch_size should error out.',\n",
       " '[SC] invalid k for predict_topk should error out',\n",
       " 'make_wheel.sh does not run cpp unit tests and error out on python unit tests',\n",
       " 'Unable to do prediction on OneshotDetection',\n",
       " 'New S3 Code Path Ignores TuriCreate SSL Runtime Config Values',\n",
       " 'tc.version_info.git_sha is empty',\n",
       " 'API Docs: tc.config page missing link for set_log_level',\n",
       " 'Can not load audio, in non-minimal package',\n",
       " 'Assertion error on creating a numpy array of an unsupported type',\n",
       " 'Missing value in Jupiter notebook visualization causes it to open in a new window and not inline',\n",
       " 'Mention in API docs and user guide that show does it inline in iPython notebook',\n",
       " 'Exception on teardown with drawing classifier',\n",
       " 'Seems to require numexpr==2.6.4',\n",
       " 'Unexpected libpng error when resizing PNG images',\n",
       " 'draw_bounding_boxes can give different colors for the same label',\n",
       " 'NDArray: Problem creating SFrame when OWNDATA is False',\n",
       " 'NDArray: Empty ndarrays become array scalars (of dangerously random value!)',\n",
       " \"[ERROR] turicreate.toolkits._main: Toolkit error: Column type of target 'rating' must be int or str.\",\n",
       " 'protocols in urls are not case insensitive.',\n",
       " 'Undefined Object',\n",
       " 'List of Lists Crash on Accordion',\n",
       " '`.plot` function exposed in SArray and SFrame',\n",
       " 'Plot APIs should ValueError on incorrect input type',\n",
       " 'Zoom not working',\n",
       " 'Row Mismatch when jumping to row + scrolling',\n",
       " 'SFrame.show has a giant weird tooltip',\n",
       " 'Get rid of disabled-key sound when we zoom into/out of `.show()` visualization',\n",
       " 'Materializing SArray... shows twice',\n",
       " 'SFrame.show in Jupyter Notebook raises exception',\n",
       " 'Build fails for want of xxd',\n",
       " 'Show() not working in master',\n",
       " 'image_analysis.load_images(...) should work with a single file',\n",
       " \"model.save() is silently failing if there's a tilde in the path\",\n",
       " 'Turicreate - Training process not proceed after 91 iteration , it loading for more than an hour @ GPU',\n",
       " \"Error when using `validation_set='auto'` in Activity classifier\",\n",
       " 'loss is nan when using beta to train detector',\n",
       " 'Text classification has wrong instruction',\n",
       " 'Import fails for 5.0b1 on macOS 10.12',\n",
       " 'Style transfer should work with 1 style image',\n",
       " 'Do not warn about .DS_Store when loading images',\n",
       " 'use try and except block around StopIteration or avoid StopIteration completely',\n",
       " 'Turi Create Visualization in 5.0b1 broken on macOS 10.13',\n",
       " 'Style transfer: make coreml model to have flexible shape input/output',\n",
       " 'Style transfer: Investigate and fix green/purple diagonal lines',\n",
       " 'Serialize all recommender models',\n",
       " 'Style transfer should set the number of iterations automatically',\n",
       " 'Empty strings showing up oddly in sketch summaries',\n",
       " 'Error reading protobuf spec. validator error: The .mlmodel supplied is of version 3, intended for a newer version of Xcode. This version of Xcode supports model version 2 or earlier.\".',\n",
       " 'Add version information to MLModel export utilities.',\n",
       " 'Save Vega JSON Broken in Beta-5.0',\n",
       " 'Loading a JSON file with an empty array should return an empty SFrame.',\n",
       " 'read_csv prints load file summary but read_json does not.',\n",
       " 'supervised learning may not always print final iteration in table printer.',\n",
       " 'ToolkitError: Unsupported PNG bit depth: 4',\n",
       " \"AttributeError: 'ConnectedComponentsModel' object has no attribute 'get'\",\n",
       " '.explore() results in blank window on Linux',\n",
       " '\"libRecommender\" Image not found error',\n",
       " 'SFrame.save(...) should auto detect .json file format',\n",
       " 'Can not export an empty SFrame',\n",
       " 'Failing unit tests in master',\n",
       " 'libRecommender should have a single dylib for all platforms',\n",
       " 'libTuriCreate.dylib does not contain the right symbols.',\n",
       " '`from builtins import str` causes Turi Create crash',\n",
       " 'Column names containing commas should be wrapped in quotes when saving to CSV',\n",
       " \"tc.object_detector.create doesn't create VNRecognizedObjectObservation ?\",\n",
       " 'Export to CoreML fails when using an arbitrary column name.',\n",
       " '.explore crashes when using a dictionary with integer keys',\n",
       " \"Can't use eGPU in Image Classification\",\n",
       " 'Broken Links in Ubuntu 14.04 Install Instructions',\n",
       " 'capi_models.cxx test of save/load fails when run in docker container',\n",
       " \"Can't get activity classification to work with MPS\",\n",
       " \"doesn't play nice with pyodbc\",\n",
       " 'DeprecationWarning: invalid escape sequence \\\\c when using nearest_neighbors.create',\n",
       " 'Activity Classifier random_split_by_session should support datasets with fewer than 100 sessions',\n",
       " 'Activity classifier should support test dataset with fewer unique labels than the training set',\n",
       " 'Activity classifier data prep computes majority, not mode',\n",
       " 'Python unit test failure in test_json',\n",
       " 'Python unit test failure in test_style_transfer',\n",
       " 'Python unit test failure in test_flexible_type',\n",
       " 'pytorch-lightning deprecations',\n",
       " 'problems with mixed precision training',\n",
       " \"Pretrained models don't work correctly on Asteroid 0.3.3\",\n",
       " 'I meet a bug when I run the ConvTasnet',\n",
       " 'CLI docs broken',\n",
       " 'Complex RNN n_layers&gt;1 is wrong',\n",
       " 'Wrong sample rate on Wham-ConvTasNet recipe when training a 16000 Hz model',\n",
       " 'Large-DCUNet-20 decoder misconfiguration',\n",
       " \"AttributeError: module 'torch.nn' has no attribute 'Identity'\",\n",
       " 'Implementing XLNet with IMDB cannot run',\n",
       " 'Potential position embedding issue in decoders.',\n",
       " 'MultiAlignedData cannot correctly return int64 data',\n",
       " 'A potential bug in transformer example',\n",
       " 'A (potential bug) in data module',\n",
       " 'Is this a bug in VAE?',\n",
       " 'pip install package on windows 10 fails with path error',\n",
       " 'Switch between model.train() and model.eval()',\n",
       " 'A bug in GPT2Tokenizer.',\n",
       " 'Cannot pickle types such as metrics',\n",
       " 'A logging bug in executor after close_files',\n",
       " 'AttentionRNNDecoder param initial_state wrapped in AttentionWrapperState',\n",
       " 'Not support Windows systemÔºü',\n",
       " \"Issue about 'int' object is not iterable\",\n",
       " 'About preparing training data',\n",
       " 'Next epoch starts after 100 iterations.',\n",
       " 'bug: VGG16 without pretrained=True option?',\n",
       " 'how to run train.ipynb?',\n",
       " 'Bug: Cannot use spot instances with hyperparameter tuning job',\n",
       " 'Error when deploying custom model via `MXNetModel.deploy`',\n",
       " 'Bug: change: use regional endpoint when creating AWS STS client #1026',\n",
       " 'Deploy method doesn\\'t work if \"tags\" attribute is supplied',\n",
       " 'Processing job: Cannot specify s3 uri as code location',\n",
       " 'SageMaker Session not created by default upon Model Class Instantiation',\n",
       " 'None always returned from `AutoML.deploy()`',\n",
       " 'Broken link to sagemaker.automl Doc',\n",
       " 'Attempt to enable data capture on an endpoint created by cloudformation will fail',\n",
       " \"`KeyError: 'ProcessingOutputConfig'` when calling Calling `ProcessingJob.from_processing_name()` for a ProcessingJob defined without a ProcessingOutput (ie. Spark Processor)\",\n",
       " 'Local _SageMakerContainer cannot run in --detached mode',\n",
       " 'training_config does not return MetricDefinitions',\n",
       " 'Upgraded pip fails to install packages with enum34',\n",
       " 'S3Downloader.list() returns malformed URIs under Windows',\n",
       " 'local_gpu with distributed training for single instance multi-gpu distributed training',\n",
       " 'Unable to suppress pandas warnings from analytics module',\n",
       " 'Default model_dir uses backslashes on Windows',\n",
       " '`session.logs_for_job` waits indefinitely for stopped jobs',\n",
       " \"bug: AttributeError: 'NoneType' object has no attribute 'module' when applying the v2 upgrade tool\",\n",
       " \"KeyError: 'NetworkConfig' in predictor.list_monitors()\",\n",
       " '[bug] incorrect sparkml-serving.json format',\n",
       " 'Using the Bring Your Own Algorithm Config with Estimator , no way currently to pass VPC configs',\n",
       " 'Using customized tensor name for prediction after deploying the model',\n",
       " 'creating estimator with role name fails if role is default sagemaker service role',\n",
       " 'AttributeError in sagemaker.local.image',\n",
       " 'Importing the sagemaker module change the logging setup',\n",
       " 'STS call on Session Object points to Global endpoint causing failure in case of Internet Disabled SageMaker notebooks',\n",
       " 'PipelineModel deploy method cannot find update_endpoint argument',\n",
       " 'Missing best config after fit for object detection fit example',\n",
       " 'About ENAS_Scheduler',\n",
       " 'Failed downloading the pretrained model of efficientnet',\n",
       " \"'OrderedDict' is not defined in classifier.py\",\n",
       " 'Fix classifier.save() and load()',\n",
       " \"Error on demo code execution -- in get_dataset if '.rec' in path: TypeError: argument of type 'NoneType' is not iterable\",\n",
       " 'Different classificationÔºåpredict classification result is sameÔºåand probability is 1       ÊåâÁÖßÂÆòÁΩëÁöÑÁ§∫‰æãÔºå‰ΩøÁî®ËÆ≠ÁªÉÂá∫ÁöÑÊ®°ÂûãÈ¢ÑÊµã‰∏§Âº†‰∏çÂêåÂàÜÁ±ªÁöÑÂõæÁâáÔºåÈ¢ÑÊµãÁªìÊûú‰∏çÁ¨¶ÂêàÈ¢ÑÊúüÔºåÁ±ªÂà´ÁªìÊûúÈîôËØØÔºåÂáÜÁ°ÆÁéá1',\n",
       " 'Does not cover the case when the list is empty.',\n",
       " \"AttributeError: 'numpy.int64' object has no attribute 'asscalar'\",\n",
       " 'Balanced accuracy error',\n",
       " 'Issue with f1 eval_metric and net.evaluate_predictions',\n",
       " 'LightGBM error special JSON characters in feature name',\n",
       " 'Tabular: os.fork() called by MXNet NN model during prediction causing OOM',\n",
       " 'Search Strategy Error Object Detection',\n",
       " '/home/nitin/anaconda3/envs/automlgluon/lib/python3.6/site-packages/sklearn/metrics/classification.py:2174: RuntimeWarning: divide by zero encountered in log   loss = -(transformed_labels * np.log(y_pred)).sum(axis=1)',\n",
       " \"'pac_score' eval_metric does not work\",\n",
       " 'How to save the model created with an ENAS_Scheduler?',\n",
       " '[HPO] ag.space.Categorical does not support keys containing `.`',\n",
       " 'Is the calculation of the RMSE of test data correct?',\n",
       " 'TabularPrediction task .fit fails on Ubuntu and Windows',\n",
       " 'MXNet segfault error when calling scheduler.run() 2+ times in same Python session',\n",
       " \"fit_summary Error KeyError: 'trial_info'\",\n",
       " 'Error when running example code for GPFIFOSearcher',\n",
       " \"ImportError: cannot import name 'TabularPrediction' from 'autogluon' (/home/tommaso/repos/gemintelligence/env/lib/python3.7/site-packages/autogluon/__init__.py)\",\n",
       " 'Initial build fails on German localized OS (prob. erroneous unit tests)',\n",
       " 'Zero size of a splitted datasets',\n",
       " 'EngineException with pytorch-native-cpu:osx-x86_64',\n",
       " 'Possible NDArray Splicing Bug',\n",
       " 'Documentation error',\n",
       " 'gradle-build java-formatter fails on unicode character (api-&gt;ProgressBar) when build run in non utf-8 system',\n",
       " 'fbprophet 0.7 breaking build',\n",
       " 'Usage of DateSplitter with max_history',\n",
       " \"NaN handling of distributions' log_prob() for samples outside distribution's support\",\n",
       " 'MQCNNEstimator behaves strangely results when constructed with distr_output',\n",
       " 'Error importing ComplexSeasonalTimeSeries on python 3.7',\n",
       " 'time_feature for Easter raised error for target date on 2018-10-06',\n",
       " 'Multiprocessing hangs when num_workers &gt; len(dataset)',\n",
       " 'Multivariate time series raise an error',\n",
       " 'canonical estimator results in shape error',\n",
       " 'Produce Json-compliant results in serve (Nan)',\n",
       " 'Bug in loading quarterly and yearly time series data',\n",
       " 'The calculation of dataset statistics reads wrong feature fields',\n",
       " 'Item metrics dataframe incorrectly formatted',\n",
       " 'Wavenet : cannot change default activation function - Gluonts 0.3.3',\n",
       " 'Incorrect number of parameters in DeepARTrainingNetwork',\n",
       " \"Can't load a serialized predictor and utilize the GPU\",\n",
       " 'DeepAR training fails for time series with Business day frequency',\n",
       " 'DeepState plotting',\n",
       " 'DeepStateEstimator with add_trend=True throws \"module \\'mxnet.symbol\\' has no attribute \\'array\\'\"',\n",
       " \"Can't save DeepState Model; RuntimeError: Cannot serialize type gluonts.distribution.lds.ParameterBounds.\",\n",
       " 'Best way to add holiday, weekend, weekend features',\n",
       " 'Binned distribution gives wrong quantiles if the probability of first bin is nonzero',\n",
       " 'Fix python version in dockerfile',\n",
       " 'PiecewiseLinear gives wrong CRPS outside of distribution support',\n",
       " 'HybridBlock to SymbolBlock conversion forgets about input/output formats',\n",
       " 'RNNEncoder: \"HybridBlock, we do not support mixed NDArrays and Symbols\"',\n",
       " 'Old models will likely not be deserializable',\n",
       " 'Tutorial notebook Exception',\n",
       " 'GluonTS sets the root logger log-level, it should not',\n",
       " 'mean_ts() has bug in class DistributionForecast & SampleForecast in forecast.py',\n",
       " 'NBeats network shares wrong parameters between blocks in sharing mode.',\n",
       " \"DeepAREstimator can't use DirichletMultinomialOutput anymore due to an assertion error\",\n",
       " 'Index of forecast is wrong in multivariate Time Series',\n",
       " 'Multiprocessing broken code',\n",
       " '\"info\" field is broken',\n",
       " \"LSTNet's scaling\",\n",
       " 'Issue in RForecast Predictor on Windows',\n",
       " 'Quick Start Example with \"NegativeBinomialOutput\" throw exception',\n",
       " 'Dynamical Feature with Different Length will Throw Exception',\n",
       " 'Dockerfiles crashing when run',\n",
       " 'Type check fails on the master branch',\n",
       " 'fix importlib deprecation warning under python3',\n",
       " 'update docker scripts',\n",
       " 'Improve error handling to report the actual error in case of an error in the custom service',\n",
       " 'Gluon pre-trained example service code is missing import statements',\n",
       " 'Trying to export model archive in same directory as model, infinite loop',\n",
       " 'None of the model endpoints appear when api-description is hit',\n",
       " 'A bug occurred while processing an error request',\n",
       " 'pip installs test/ and contrib/ directories',\n",
       " 'hypotheses lengths off by one',\n",
       " 'No error thrown when training with source factor files but no validation source factor files given',\n",
       " 'Release 2.1.16: metrics file loses a line every time training is stopped and then continued',\n",
       " 'Incompatibility  &gt;=2.1.17 vs. &lt;=2.1.16',\n",
       " 'Flaky Tests: HTTPSource',\n",
       " 'LightGBMClassifier: for multiclass objective, wrong probability and prediction',\n",
       " 'java.net.ConnectException: Connection refused (Connection refused) with LightGBMClassifier in Databricks',\n",
       " 'Support min_gain_to_split and max_delta_step',\n",
       " '\"port out of range error\" when training',\n",
       " 'Error - LightGBMClassifier during train with initScoreCol for multi-label classification:  java.lang.ClassCastException: org.apache.spark.ml.linalg.SparseVector cannot be cast to java.lang.Double',\n",
       " '[LightGBM] Train Lambdamart failed with \"org.apache.spark.shuffle.MetadataFetchFailedException: Missing an output location for shuffle 1\"',\n",
       " 'Issues Fitting \"Same\" Data into Light GBM Classifier',\n",
       " 'autogenerated requirements.txt is missing version numbers of select dependencies',\n",
       " 'bentoml serve fails with error is built / packaged on Windows and deployed on Linux',\n",
       " 'Configurable max request body size',\n",
       " 'Cli get command crash if bento name is not found bento meatadata store',\n",
       " 'Failed to set suitable adapter on DefaultOutput, when returning InferenceResult as result',\n",
       " 'Cannot retrieve BentoService from S3 Minio',\n",
       " 'OnnxModelArtifact should work when using onnxruntime-gpu package',\n",
       " 'Cannot delete bento that was created with same name:version as an older (deleted) bento',\n",
       " 'Document Formatting error on the Getting Started page',\n",
       " '[Bug]Incorrect endpoint API description on swagger UI',\n",
       " '--enable-microbatch got pickle error',\n",
       " 'No bundled_pip_dependencies in building container',\n",
       " 'bentoml retrieve fails after upgrading to 0.10.1',\n",
       " 'Problem accessing gunicorn server from kubernets ingress',\n",
       " '`bentoml lambda list` failed with PostgreSQL 11',\n",
       " '`request_id` in response header seems broken',\n",
       " 'Failure to containerize model if build-arg value contains multiple `=` characters',\n",
       " 'BentoService#save should copy local python modules',\n",
       " \"Cannot use list in ImageHandler's input_name\",\n",
       " 'AWS Lambda Deployment: error creating bentoml home dir and config file',\n",
       " 'Failing to create and API on Windows',\n",
       " \"TypeError: __init__() got an unexpected keyword argument 'case_sensitive'\",\n",
       " 'KerasModelArtifact#pack expects type: keras.engine.training.Model',\n",
       " \"AttributeError: module 'fastai' has no attribute 'basic_train'\",\n",
       " 'Python packaged model implicitly requires bentoml package',\n",
       " 'Failed deployments to sagemaker cannot be deleted from deployments list',\n",
       " 'Using curl to test lambda deployment fails with dataframe handler',\n",
       " 'Problem importing handlers from external files using bentoml serve',\n",
       " 'Python code dependencies not saved in SavedBundle',\n",
       " 'botocore.exceptions.ClientError',\n",
       " \"YataiService's Docker doesn't include psycog\",\n",
       " 'Error in execution of \"bentoml get\" cli command',\n",
       " \"The CLI 'bentoml delete' deletes the saved bundle, but shows an error message\",\n",
       " 'got internal server error when trying to invoke sagemaker endpoint',\n",
       " 'Error when creating custom handler class',\n",
       " '--enable-microbatch raises error in 0.8.1 with XGboost example',\n",
       " '--enable-microbatch breaks in text files with \",\" in it',\n",
       " 'TfTensorOutput casts floats as ints',\n",
       " '\"OSError address already in use\" when running \"bentoml serve\" with --enable-microbatch',\n",
       " 'Permission issue while build docker image',\n",
       " 'DataFrame Input - pass NaN values in JSON Request',\n",
       " 'Yatai does not handle the STS assumed role in operator.get_arn_role_from_current_aws_user()',\n",
       " 'Testing `Content-Type` directly is problematic in JsonInput adapter',\n",
       " \"' npm run build' command failed while running BentoML Web UI\",\n",
       " \"'yarn start ' and 'npm start'  failing while loading web UI\",\n",
       " 'When I built a H2O-based image, the following error occurred',\n",
       " 'Failed to package local pip installed dependency in 0.8.4',\n",
       " 'cli throw RecursionError when using --enable-microbatch',\n",
       " 'DataframeInput does not handle indices',\n",
       " 'After closing project, ann mlconfig file is not consitant',\n",
       " 'Confusion Matrix Dialog has wrong caption for validation set',\n",
       " 'Encoder in environment.GymEnvironment',\n",
       " \"Prediction on FPGA doesn't work without -cache option\",\n",
       " 'OUTPUT_DIR with the gcs path does not work',\n",
       " 'An error happens with no detected bounding box on object detection',\n",
       " 'Incompatible version `absl-py`',\n",
       " 'In case of really small dataset. MAX_STEPS can be less than 1 and lead to error.',\n",
       " 'LMBiSeNet inference test error',\n",
       " 'Fix round',\n",
       " 'DLK tests for segmentation fail on Arm with hard_quantize=False',\n",
       " 'TF1.15.0 has AddV2, TF1.14.0 had Add - compiler throws error',\n",
       " 'Cannot make libdlk_x86_avx.a',\n",
       " 'Dataset directory issue in Keypoint Detection',\n",
       " 'Variables declaration are missing in usb_camera_demo.py',\n",
       " 'Cannot run blueoil init with  enable data augmentation? : No',\n",
       " 'SSDRandomCrop augmentation option  is not working in classification tutorial.',\n",
       " 'lm_fpga.elf sometimes fails with classification',\n",
       " 'UserWarning for LmSegnetV1Quantize',\n",
       " 'GapClassificationDM giving this error',\n",
       " 'No output for your own images also.',\n",
       " 'Error Continued...........',\n",
       " 'Chars_CZ, Chars_En undefined?',\n",
       " 'wordClass3 model (CTC) in OCR Evaluator is not working.',\n",
       " 'Error while training the model',\n",
       " 'Error:while executing ocr_evaluator.ipynb',\n",
       " 'WriteProtoToTextFile writes nothing when the target file does not exist',\n",
       " \"ByteScheduler: 'ByteCore' object has no attribute 'set_broadcaster' in Pytorch plugin\",\n",
       " 'Crash during MXNet distributed training',\n",
       " 'Missing id() op in bytescheduler.torch.optimizer.py',\n",
       " 'undefined symbol during import',\n",
       " 'mxnet distributed trainer broadcast leads to corrupted model',\n",
       " \"BytePS w/ MXNet doesn't work w/o docker container\",\n",
       " 'Different memory consumption among GPUs',\n",
       " 'Lost parameters in logs_path/configs/_config.json',\n",
       " '[fix] fix registry exception for duplicate factory',\n",
       " 'Fix in BaseCheckpointCallback `_metrics` not only for the last stage but for all',\n",
       " 'Flake8 error while executing \"make codestyle\" command',\n",
       " 'imread fails with non-jpg files',\n",
       " \"Specifying cuda device through SupervisedRunner doesn't get picked up\",\n",
       " 'AssertionError: wrong output size (1D expected) when using PrecisionRecallF1ScoreCallback for Segmentation',\n",
       " 'Incorrect Tensorboard import',\n",
       " 'Repair resume: resume epoch and stage(!)',\n",
       " \"AttributeError: 'numpy.ndarray' object has no attribute 'items' when using `MulticlassDiceMetricCallback`\",\n",
       " 'Bug in EarlyStoppingCallback with minimize=False',\n",
       " 'SupervisedWandbRunner logs wrong number of epochs to WandB',\n",
       " 'text2embedding bug',\n",
       " \"AttributeError: module 'catalyst.utils' has no attribute 'detach' while running Segmentation Tutorial\",\n",
       " 'Your .dependabot/config.yml contained invalid details',\n",
       " 'WandbLogger bug',\n",
       " 'fix docker-deploy action',\n",
       " 'Bug: catalyst depends from system git',\n",
       " 'OneCycleLRWithWarmup does not starts with init_lr',\n",
       " 'EarlyStoppingCallback considers first epoch as bad',\n",
       " 'Wrong order of embeddings in CMC score callback',\n",
       " 'Warning in torch.lr_scheduler',\n",
       " 'read_squad_examples() does not work with our custom input object',\n",
       " \"NameError: name 'device' is not defined in predict() method\",\n",
       " 'nptying in setup.py',\n",
       " 'Doc test fails at optimizer.py',\n",
       " 'MultiprocessingIterator Deadlock',\n",
       " 'Error in F.softmax when using non-contiguous array',\n",
       " 'Documentation bug report: Some of the descriptions in the tables are truncated.',\n",
       " 'Doc: Color highlighting is lost in chainer.functions.reshape code example',\n",
       " \"VariableNode's dtype mismatch if Variable.data is manually assigned\",\n",
       " 'chainermn.links.create_mnbn_model raises an assertion error for functions in chainer.Sequential',\n",
       " 'Performance of ACER compared to A3C',\n",
       " 'Openpose build for Unity',\n",
       " \"nvcc fatal : Unsupported gpu architecture 'compute_71' CMake Error at openpose_generated_renderHand.cu.o.Release.cmake:219\",\n",
       " 'Failed to run the exmple:\"2_thread_user_input_processing_output_and_datum.bin\"',\n",
       " 'Crash in demo app when doing 3D reconstruction with two FLIR cameras (Windows)',\n",
       " 'Openpose 1.6.0 Windows Demo frame_rotate limitation/bug?',\n",
       " 'Execution error on Windows 10',\n",
       " 'GTX980,  Cuda check failed (9 vs. 0)  invalid configuration argument',\n",
       " 'Latest master (ce62c53) openpose demo crashes on images (Ubuntu16.04, CUDA8, OpenCV3.1)',\n",
       " 'OpenPoseConfig.cmake includes openpose and caffe folders',\n",
       " 'int should be replace with float in poseTriangulation.cpp',\n",
       " 'failed to run the examples in the tutorial_api_cpp form 4 to 9',\n",
       " '[Bug] Lazy Tensor Internal Indexing Error for Large Variance Evaluation (MultitaskMultivariateNormal)',\n",
       " '[Bug] ConstantMulLazyTensor can only add batched tensors',\n",
       " '[Bug] Fantasizing is very slow on GPU due to torch.qr',\n",
       " 'Dimension out of range error',\n",
       " '[Bug] BlockDiagLT symeig fails when no eigenvectors are desired',\n",
       " '[Bug]',\n",
       " 'Fast pred var? More like 1/3 as fast pred var',\n",
       " 'Where is `fast_pred_var` moving?',\n",
       " 'batch evaluation of Hadamard Multi-Task GP fails',\n",
       " 'Error in `psd_safe_cholesky` (when data has low noise?)',\n",
       " '[Bug] TypeError when calling `backward` on `gpytorch.functions.logdet`',\n",
       " \"AttributeError: 'LazyEvaluatedKernelTensor' object has no attribute 'lazy_tensors'\",\n",
       " '[Bug] Upstream changes to tensor comparisons breaks things',\n",
       " '[Bug] Invalid index in gather during call to mll in training loop',\n",
       " \"Completion doesn't work with zsh\",\n",
       " 'Adding certain pip and/or conda packages removes pre-installed packages',\n",
       " 'Tags are not propagated to API log groups',\n",
       " 'GPU nodes are being overprovisioned during scale up',\n",
       " 'Bad request is returning 500 status code instead of 400',\n",
       " 'Verify accuracy of request metrics for a local deployment using multiple threads',\n",
       " 'Enable CORS for APIs accessed via API Gateway route',\n",
       " 'Image download shown as successful but see error: \"No such image: cortexlabs/python-predictor-cpu:0.18.1\"',\n",
       " 'Fix scalar and no-rank/no-shape tensor validation when using the TensorFlow predictor',\n",
       " 'Environments get overridden when cortex cluster up commands run in parallel',\n",
       " \"Can't remove a job with one worker using BatchAPI and DELETE request\",\n",
       " 'Submitted job is stuck in running state',\n",
       " 'Local models not available when running cortex get for the TF predictor and when live-reloading is enabled',\n",
       " 'API tries to download local model from S3 (for TensorFlow, MMC)',\n",
       " 'Model names with same prefix breaks API deployment',\n",
       " 'Cloudwatch logs get filtered away',\n",
       " 'Pods occasionally error on first attempt',\n",
       " 'Fix Python Packages',\n",
       " 'Show TF Serving logs before API logs',\n",
       " 'Log line format',\n",
       " 'Iris Keras example crashes',\n",
       " 'Prevent out-of-memory from taking down nodes',\n",
       " \"API doesn't get updated if the request handler path is changed\",\n",
       " \"API gets redeployed if project changes even when a request handler hasn't been specified\",\n",
       " 'API /signature endpoint is not reachable',\n",
       " 'Example for tensorflow is not present',\n",
       " 'Log stream quits when scaling worker node group from 0 to 1',\n",
       " 'Improve cluster-config validations',\n",
       " 'Intermittent failed requests during rolling updates',\n",
       " 'Inconsistency definition of the training param of load_trained_model_from_checkpoint function',\n",
       " 'LayerNorm excluded from weight decay',\n",
       " 'Bug with Apollo refreshing JWT token after some time away',\n",
       " 'Filter section should be scrollable by dragging custom scrollbar',\n",
       " 'Tensorflow \"Illegal instruction\" on some machines',\n",
       " 'Nginx infinite loop',\n",
       " 'Exif rotation data not being used when first importing and determining image width/height',\n",
       " \"convert.py doesn't work with OriginalHighRes trainer\",\n",
       " \"Extract with dlib-cnn doesn't work on macOS\",\n",
       " \"Deleting a nested layer/connection doesn't refresh the nested visualizer\",\n",
       " 'Backwards compatibility with old projects w/o operation color',\n",
       " 'Assigning value to const when null pointer (v2.0.0)',\n",
       " 'Exported with incorrect static input',\n",
       " 'Invalid python when exporting pipeline w/o outputs',\n",
       " 'Setting data input should not scrub data type (v2)',\n",
       " \"Plots don't always show up (race condition?)\",\n",
       " 'Output operation not saving the artifact in the correct location',\n",
       " 'Removing library should remove initCode node for the given library',\n",
       " 'Action button not updated after removing library',\n",
       " 'Installing extension fails \"Could not find project in current or any parent directories\"',\n",
       " 'Failed pipeline execution with operation outputs',\n",
       " 'Error when creating layer in nn architecture',\n",
       " 'Update to latest version of webgme-easydag',\n",
       " \"Can't create a node in the pipeline\",\n",
       " \"ForwardViz not compatible with WebGME's DefaultLayout\",\n",
       " 'Incorrect argument order when executing pipeline snapshot (not debug mode)',\n",
       " 'Export CLI fails when pipeline name has spaces',\n",
       " 'DeepForge extensions installation fails on windows',\n",
       " 'Blob retrieval console errors when using ArtifactIndex',\n",
       " 'Set default visualizer for custom utils to TabbedTextEditor',\n",
       " 'Operation attributes can lost type info on removing default value',\n",
       " 'Better name for operation copies',\n",
       " 'CheckUpdates error in server log on pipeline (seed) updates available',\n",
       " 'Update docs on custom operations',\n",
       " 'Invalid reference options are being loaded after adding deepforge-keras',\n",
       " 'Resource Index should have generic template for more than architectures',\n",
       " \"FAB toasts don't disappear when creating new nodes\",\n",
       " 'Awkward time duration display format',\n",
       " 'Graphs not reused on subsequent executions',\n",
       " 'matplotlib missing from worker docker image',\n",
       " 'Invalid Connections not being cleaned up automatically',\n",
       " 'Better error handling for `.content of undefined`',\n",
       " '\"Metadata about result types not found.\" when process exits involuntarily',\n",
       " 'Tests failing - cannot find ComponentSettings',\n",
       " 'Unused deepforge-keras dependency in package.json',\n",
       " 'Cached nodes not updated on SYNCED save result',\n",
       " 'enableJobCaching option (in components.json) is ignored',\n",
       " 'Footer, Header should be secondary visualizers',\n",
       " '`define is not defined` when running pipeline',\n",
       " 'Filter out additional Job attributes from UI',\n",
       " 'Error on export pipeline',\n",
       " \"Error if change to branch where active node doesn't exist\",\n",
       " 'ImportArtifact fails when only one storage backend exists',\n",
       " 'Storage clients fail in NodeJS',\n",
       " 'GME Blob Storage Backend missing config metadata',\n",
       " 'Upload non-existent hashes when checking for updates',\n",
       " 'Incorrect semantic versioning comparison when checking updates',\n",
       " 'Ensure correct fork name if execution forks immediately',\n",
       " 'Update project seed to include latest pipeline library',\n",
       " 'Exception on ArtifactIndex after running pipeline',\n",
       " 'architecture not showing up when nn is library (nondev)',\n",
       " 'GenerateJob fails with null active node',\n",
       " 'Unable to delete running execution',\n",
       " 'Download button for data not showing',\n",
       " 'Extra console.log left in start.ejs',\n",
       " 'Log stderr and stdout in local execution',\n",
       " 'Set DEEPFORGE_ROOT in local compute backend',\n",
       " 'Error on export pipeline',\n",
       " 'Compute dropdown below options for specific backend',\n",
       " 'Storage options are incorrect',\n",
       " '`outputs` not defined when executing Input operations',\n",
       " '`setAttribute` not defined for ExecutePipeline',\n",
       " 'Running Execution Deletion throws error in the server',\n",
       " 'Storage backend not found: SciServerFiles',\n",
       " 'Duplicate logs in Output node',\n",
       " 'GME Storage backend incompatible with SciServer compute backend',\n",
       " '\"Metadata about result types not found\" when running job with multiple inputs',\n",
       " 'ExecutionIndexWidget Throws an Error for Failed Executions',\n",
       " 'forEach of undefined when missing metadata',\n",
       " 'ImportArtifact errors on upload',\n",
       " 'ImportArtifact should import to the artifacts directory for the given project',\n",
       " 'Incorrect error in storage client base class',\n",
       " 'Remove utils.build.js from version control',\n",
       " 'Error Connecting to server when using HTTPS',\n",
       " 'Ensure storage adapter metadata cannot be mutated',\n",
       " 'Extra logs printed with server execution',\n",
       " 'Remove double slash path in execution files',\n",
       " 'Restarting executions throws an exception',\n",
       " 'PipelineEditor incorrect connection type',\n",
       " 'Bad formatting for subfigure titles',\n",
       " 'Errors when creating/editing metadata',\n",
       " 'core does not have method getAttributeNames (TwoPhaseCore)',\n",
       " \"Cannot read property 'title' of undefined (PlotlyDescExtractor)\",\n",
       " 'Deleting last remaining Execution in ExecutionIndex throws an error',\n",
       " 'Set text size/theme in the log viewer',\n",
       " 'Ace error thrown in log viewer',\n",
       " 'Cannot set NODE_ENV in docker container',\n",
       " 'Operation name reset on editing code',\n",
       " 'Cannot call `getValidAttributeNames` of null',\n",
       " 'Config is null error when switching to the Custom Utils tab',\n",
       " 'Importing deepforge libraries fails for non-guest users',\n",
       " 'Invalid branch name when importing library',\n",
       " 'Missing environment.yml file error in CLI',\n",
       " 'Set up git lfs on travis',\n",
       " 'Config can still be null for TabbedTextEditor',\n",
       " 'Flaky integration tests in CI',\n",
       " 'Unable to uncheck execution by clicking checkbox',\n",
       " 'Only running CondaUtils tests',\n",
       " 'SciServer tokens are cached indefinitely',\n",
       " 'Base conda environment creation failing on SciServer',\n",
       " 'SciServer Compute file upload fails',\n",
       " 'aws-s3-sdk requirejs path is overwritten on new webgme component',\n",
       " 'Unexpected end of JSON input for metadata info',\n",
       " 'Flaky storage features tests',\n",
       " 'Graphs do not fill all of the available space',\n",
       " 'Conda worker environment not activated if no job dependencies specified',\n",
       " 'Issues with worker environment in #1586',\n",
       " 'Out of date library when creating new project',\n",
       " 'Heartbeat \"error: unauthorized\" when running pipeline',\n",
       " 'RootViz changes when leaving the screen',\n",
       " 'Fix Compose File location in deploy-deepforge and cause ssh action to fail on first error.',\n",
       " '\"Illegal instruction\" when using tensorflow on SciServer Compute',\n",
       " 'Extra logs displayed when running jobs on SciServer Compute',\n",
       " 'Remove deepforge-keras from package.json',\n",
       " 'Downloading jq from github in dev entrypoint is sometimes stuck',\n",
       " 'BlobClient detecting NaN for the server port',\n",
       " 'config mount in deployment overwrites extension changes',\n",
       " 'GME storage is unauthorized when user accounts enabled',\n",
       " 'Error loading config with new inputs',\n",
       " 'Error when clearing SciServer auth tokens',\n",
       " '\"Pickle data was truncated\" error',\n",
       " 'Data upload fails when connecting Input-&gt;Output w/ different storage backends',\n",
       " '\"Data retrieval failed for ___: [object Response]\"',\n",
       " 'Better error logging for SciServer Files adapter',\n",
       " 'Error when loading config dialog without and saved settings',\n",
       " 'CoreIllegalArgumentError when running pipeline',\n",
       " 'Authorization credentials not encrypted for pipeline inputs',\n",
       " 'npm start fails if conda not available',\n",
       " 'Error in FigureExtractor.js while the node is null',\n",
       " 'Closing ConfirmDialog should return \"false\"',\n",
       " 'Remove download button in DataEditor view',\n",
       " \"Don't try to set special operation attributes in interface editor\",\n",
       " 'GME library releases not adding new line to releases.jsonl',\n",
       " 'PipelineEditor addNode Operation bad names',\n",
       " 'forAllNodeChanges fails if node created in later staged changes',\n",
       " 'Job status should be \"canceled\" after calling \"cancelJob\"',\n",
       " '\"Download\" button broken for artifacts',\n",
       " 'Error in docker command in the quick start docs',\n",
       " 'Check for Metadata nodes in FigureExtractor',\n",
       " 'Ensure registered actions are not removed until unregistered',\n",
       " 'Interactive editor should unregister actions on destroy',\n",
       " 'Switching backends in matplotlib 3.3.0 fails',\n",
       " \"PlotlyGraph Widget's Container's height defaults to zero\",\n",
       " 'InteractiveCompute missing from webgme-setup.json',\n",
       " 'Hide \"provenance\" attribute from operation interface editor',\n",
       " 'Compute tabs not switching on second open',\n",
       " 'Ace editor worker not destroyed on viz change',\n",
       " 'Unable to add artifacts to interactive session',\n",
       " 'Loading artifacts can result in duplicate entries in TensorPlotter',\n",
       " 'TensorPlotter plot is not responsive to window resizes',\n",
       " 'Pipeline not starting on SciServer Compute',\n",
       " 'SciServer jobs showing as failing',\n",
       " 'Pipeline name not updating on changing branch',\n",
       " 'Plotly error on pipeline restart',\n",
       " 'Bad error message from TwoPhaseCore w/ null node',\n",
       " 'Incorrect plugin metadata for UploadSeedToBlob',\n",
       " ...]"
      ]
     },
     "execution_count": 58,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "BRsummary"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1c462c14",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f1ec7a06",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
